lib/v2/rattibha/user.js

```diff
@@ -28,15 +28,17 @@ module.exports = async (ctx) => {
     // Get tweet full text
     const items = await Promise.all(
         list.map(async (item) => {
+            item.threads = [];
             const threadPromises = [item.threadData1URL, item.threadData2URL].map((url) =>
-                ctx.cache.tryGet(url, async () => {
+                ctx.cache.tryGet(item.link, async () => {
                     const { data } = await got(url, { headers: { accept: 'application/json' } });
-                    return data;
+                    const threads = data.tweets ? data.tweets : data;
+                    item.threads = [...item.threads, ...threads];
+                    return item;
                 })
             );
-            const threadsArray = await Promise.all(threadPromises);
-            const threads = [...threadsArray[0].tweets, ...threadsArray[1]];
-            item.description = threads.reduce((accumulator, tweet) => `${accumulator}${tweet.tweet_detail.info.text}<br>`, '');
+            await Promise.all(threadPromises);
+            item.description = item.threads.reduce((accumulator, tweet) => `${accumulator}${tweet.tweet_detail.info.text}<br>`, '');
             return item;
```

Comment:
https://github.com/DIYgod/RSSHub/pull/12633#discussion_r1224226271

I mean caching this `item` at line 42 (the returned `item`)